GOVERNANCE.md

```diff
@@ -1,45 +1,120 @@
+
+Currently SIG membership requires invitation - potential qualified members are Active Contributors in a particular module. It is our intention to properly guide or mentor community talents, and help them advance to the higher-level Reviewer, Committer and Maintainer within an SIG.
+
+Each SIG must have a charter that specifies its scope (topics, code repositories and directories), responsibilities, areas of authority, how members and roles of authority/leadership are selected/granted, how decisions are made, and how conflicts are resolved. See the SIG charter process for details on how charters are managed. SIGs should be relatively free to customize or change how they operate, within some broad guidelines and constraints imposed by cross-SIG processes.
+
+See [sig governance](/sig-governance/sig-governance.md) for more details about current SIG operating processes and mechanisms.
+
+#### Working Groups
+
+A Working Group is formed by community developers who gather together to accomplish a specific goal. To achieve the goal, some WGs may span over multiple SIGs, and some WGs may only focus on something specific in a specific SIG.
+
+Each WG has its life cycle. Once the goal is completed, the group will be disbanded. The only goal of WG operations and management is to ensure that the goals set by the group are completed at the right time. In general, the working group hold periodic meetings to summarize the current project progress and determine the plans for next steps.
+
+<!---to add Working Group governance link -->
+
+## Contributions
+
+Anyone can contribute to the project, regardless of their skills, as there are many ways to contribute. The various ways of contributing are described in more detail in a separate document. For details, see [Contributing to TiKV](https://github.com/tikv/tikv/blob/master/CONTRIBUTING.md).
+
+## Decision making and voting
+
+Decisions are build on consensus between PMC members or Maintainers. Depending on the nature, decisions about the future of the project are made by the PMC or by Maintainers. As the technical authority, Maintainers set technical direction and priorities for the sub-project, while PMC make other high-level decisions like establishment and operation policies of sub-projects, sub-structures,promoting new Maintainers, security processes, etc.
+
+New proposals and ideas (changes of major features, organization, or processes) can be brought to the PMC or Maintainers’ attention through the [Request for Comments (RFC)](https://github.com/tikv/rfcs) process. For the change to happen, the RFC must earn the supermajority (2/3) votes in the corresponding group.
```

Comment:
This includes **all** major changes, those the PMC or maintainers are wanting to enact, or those our contributors want to push, yes?

So all new `Feature` tagged PRs to any TiKV owned project needs an RFC?

Reply:
Isn't it our current practice to submit RFC for any new features to be developed?

Reply:
I want to make sure this is a "No exceptions" rule. While technically it's our practice we don't follow it religiously.

Reply:
Yeah I get what you mean. It's also my intention to state things as clear as possible so that there is less room not to follow them